modules/sandboxed-containers-deleting-operator.adoc

```diff
@@ -0,0 +1,31 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-operator_{context}"]
+= Deleting {sandboxed-containers-operator}
+
+You can delete the {sandboxed-containers-operator} from your cluster.
+
+.Prerequisites
+
+* You have {sandboxed-containers-first} 1.0.0 installed.
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Delete the {sandboxed-containers-operator} subscription from Operator Lifecyle Manager (OLM) by running the following command:
++
+[source,terminal]
+----
+$ oc delete subscription openshift-sandboxed-containers-subscription -n openshift-sandboxed-containers
+----
+
+. Set the cluster service version (CSV) name for {sandboxed-containers-first} as an environment variable by running the following command:
++
+[source,terminal]
+----
+ $ CSV_NAME=$(oc get csv -n openshift-sandboxed-containers -o=custom-columns=:metadata.name)
```

Comment:
What is this step supposed to do. @jensfr, does that make sense to you?

Reply:
It does nothing as written as the namespace is wrong.  Should be "openshift-sandboxed-containers-operator" to do anything useful.  Regardless, it is unclear what is supposed to be done with $CSV_NAME

Reply:
@cpmeadors, @c3d Should this be a part of the documentation? If so, what should this be replaced with?

Reply:
@jensfr Could you please comment? Is this an important verification step? If so, should we simply rewrite it as 

```
oc get csv -n openshift-sandboxed-containers-operator
```

?

Reply:
We are missing a step in addition to correcting the namespace as @cpmeadors pointed out: 

Deleting the CSV
`oc delete csv ${CSV_NAME} -n openshift-sandboxed-containers-operator`

Reply:
@jensfr